xbmc/guilib/GUIImage.dox

```diff
@@ -71,6 +72,7 @@ important, as `xml` tags are case-sensitive.
 | info          | Specifies the information that this image control is presenting. Kodi will select the texture to use based on this tag. [See here for more information](http://kodi.wiki/view/InfoLabels).
 | fadetime      | This specifies a crossfade time that will be used whenever the underlying <b>`<texture>`</b> filename changes. The previous image will be held until the new image is ready, and then they will be crossfaded. This is particularly useful for a large thumbnail image showing the focused item in a list, or for fanart or other large backdrops.
 | background    | For images inside a container, you can specify background="true" to load the textures in a background worker thread. [See here for additional information about background loading](http://kodi.wiki/view/Background_Image_Loader).
+| imagefilter   | This can take two values: <b>`linear`</b> or <b>`nearest`</b>. Linear scaling is the default, and smoothly blurs between pixels. Nearest neighbor scaling will apply a "pixelated" effect to the image.
```

Comment:
mind adding the revision tag a bit below just like the `infill` one? Sorry for starting the new year nudging :)